gwlfe/gwlfe.py

```diff
@@ -16,18 +18,692 @@
+    # AnnSumSSOrgs = np.zeros((50, 12))
+    # AnnSumUrbOrgs = np.zeros((50, 12))
+    # AnnSumWildOrgs = np.zeros((50, 12))
+    # AnnSumTotalOrgs = np.zeros((50, 12))
+    # AnnSumCMStream = np.zeros((50, 12))
+    # AnnSumOrgConc = np.zeros((50, 12))
+    # AnnSumLuRunoff = np.zeros((50, 25))
+    # AnnSumLuErosion = np.zeros((50, 25))
+    # AnnSumLuSedYield = np.zeros((50, 25))
+    # AnnSumLuDisNitr = np.zeros((50, 25))
+    # AnnSumLuTotNitr = np.zeros((50, 25))
+    # AnnSumLuDisPhos = np.zeros((50, 25))
+    # AnnSumLuTotPhos = np.zeros((50, 25))
+    # AnnSumSepticNitr = np.zeros(50)
+    # AnnSumSepticPhos = np.zeros(50)
+    # AnnSumTotAnimalN = np.zeros(50)
+    # AnnSumTotAnimalP = np.zeros(50)
+    # AnnSumTotTileDrainSed = np.zeros(50)
+    # AnnSumTotTileDrainN = np.zeros(50)
+    # AnnSumTotTileDrainP = np.zeros(50)
+    # AnnSumTotGroundNitr = np.zeros(50)
+    # AnnSumTotGroundPhos = np.zeros(50)
+    # AnnSumAnnPointNitr = np.zeros(50)
+    # AnnSumAnnPointPhos = np.zeros(50)
+    # AnnSumTotStreamBankEros = np.zeros(50)
+    # AnnSumTotStreamBankN = np.zeros(50)
+    # AnnSumTotStreamBankP = np.zeros(50)
+
     ReadGwlfDataFile.ReadAllData()
+
+    # ########################- BEGIN MAIN GWLF RUN CODE -########################
+    # #>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>><<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<#
+    # #>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>><<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<#
+    # ############################################################################
+
+    # CALCLULATE PRELIMINARY INITIALIZATIONS AND VALUES FOR
+    # WATER BALANCE AND NUTRIENTS
     PrelimCalculations.InitialCalculations()
 
-    for year in range(100):
-        for month in range(12):
-            for day in range(30):
-                CalcCnErosRunoffSed.CalcCN()
+    # MODEL CALCULATIONS FOR EACH YEAR OF ANALYSIS - WATER BALANCE,
+    # NUTRIENTS AND SEDIMENT LOADS
+
+    # TODO: Placeholder declaration
+    NYrs = 10
+
+    for y in range(0, NYrs):
+        # TODO: Placeholder declarations.
+        MonthPondNitr = np.zeros(12)
+        MonthPondPhos = np.zeros(12)
+        MonthNormNitr = np.zeros(12)
+        MonthShortNitr = np.zeros(12)
+        MonthShortPhos = np.zeros(12)
+        MonthDischargeNitr = np.zeros(12)
+        MonthDischargePhos = np.zeros(12)
+
+        # FOR EACH MONTH...
+        for i in range(0, 12):
+
+            # Initialize monthly septic system variables
+            # TODO: We many not need to initialize these.
+            # Consider removing once these variables are
+            # populated with real data.
+            MonthPondNitr[i] = 0
+            MonthPondPhos[i] = 0
+            MonthNormNitr[i] = 0
+            MonthShortNitr[i] = 0
+            MonthShortPhos[i] = 0
+            MonthDischargeNitr[i] = 0
+            MonthDischargePhos[i] = 0
+
+            # LOOP THROUGH NUMBER OF LANDUSES IN THE BASIN TO GET QRUNOFF
+            # TODO: Placeholder declarations.
+            NLU = 12
```

Comment:
I wonder why this is 12... There are 10 rural land uses and 6 urban land uses for a total of 16.